### PR TITLE
Remove types, utility code, etc relating to digital subscriptions

### PR DESCRIPTION
**Pull request by marialani**
## What are you doing in this PR?
Following the removal of client-side code for the digital subscription landing and checkout pages https://github.com/guardian/support-frontend/pull/4741, this removes or refactors additional client side code including types, utility code, mock data, react components and css styling previously used on these pages.  

[**Trello Card**](https://trello.com/c/t2LoxA43/1072-remove-types-utility-code-etc-relating-to-digital-subscriptions)

## Why are you doing this?
Since the digital subscription pages have been deprecated we no longer need a lot of this code.

### CODE REVIEW

**Comment by marialani** [comment]
[Remove types, utility code, etc relating to digital subscriptions](https://trello.com/c/t2LoxA43/1072-remove-types-utility-code-etc-relating-to-digital-subscriptions)

File: support-frontend/assets/helpers/timeBoundedCopy/timeBoundedCopy.tsx [hunk 1]
```diff
@@ -13,23 +13,11 @@ type TimeBoundCopy = {
...
-						digital subscription gives you the richest experience of Guardian
-						journalism. It also sustains the independent reporting you love.
-					</p>
-					<p>
-						Plus, for a limited time, get inspiration for new ideas to improve
-						your life in our exclusive special edition, Self and Wellbeing.
-					</p>
-				</>
-			),
+			copy: <></>,
```
Comment (undergroundquizscene) [inline support-frontend/assets/helpers/timeBoundedCopy/timeBoundedCopy.tsx:20]: What’s this? If `stopShowingOn` being in the past means it’s no longer used, can we just get rid of it altogether?

**Review by undergroundquizscene — APPROVED** [review]
Looks good!

I remember there being something about needing to leave some part of digi subs in place for CSRs to be able to switch people onto digi subs (or something like that). Is that still the case, and is any of this frontend stuff needed for that? I don’t have the clearest idea and I know this is pretty vague, but I figured it’s the kind of thing that I should mention just in case.

File: support-frontend/assets/helpers/timeBoundedCopy/timeBoundedCopy.tsx (hunk 1 above)
Comment (marialani) [reply support-frontend/assets/helpers/timeBoundedCopy/timeBoundedCopy.tsx:20]: We could definitely get rid of this variable but in the past we've kept an example in of how to structure the timed copy data. This is a helper (not currently in use) to allow us to easily make copy changes on our landing pages.

**Comment by marialani** [comment]
> Looks good!
> 
> I remember there being something about needing to leave some part of digi subs in place for CSRs to be able to switch people onto digi subs (or something like that). Is that still the case, and is any of this frontend stuff needed for that? I don’t have the clearest idea and I know this is pretty vague, but I figured it’s the kind of thing that I should mention just in case.

Thanks for flagging! I will raise this tomorrow just in case there is something I may have pulled out that may still be needed.

File: support-frontend/assets/helpers/timeBoundedCopy/timeBoundedCopy.tsx (hunk 1 above)
Comment (undergroundquizscene) [reply support-frontend/assets/helpers/timeBoundedCopy/timeBoundedCopy.tsx:20]: Ah yes, that makes sense – thanks for the explanation :)

File: support-frontend/assets/components/subscriptionCheckouts/thankYou/returnSection.tsx [hunk 2]
```diff
@@ -16,11 +19,11 @@ const buttonStyles = css`
 	}
 `;
 type PropTypes = {
-	subscriptionProduct: SubscriptionProduct;
+	subscriptionProduct: SubscriptionProduct | typeof DigitalPackAddOn;
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/components/subscriptionCheckouts/thankYou/returnSection.tsx:22]: Is there a reason we didn't add DigitalPackAddOn to type SubscriptionProduct (used quite widely so would have to change a lot?)

File: support-frontend/assets/pages/subscriptions-redemption/subscriptionsRedemption.tsx [hunk 3]
```diff
@@ -23,6 +23,7 @@ const thankyouProps = {
 	countryGroupId,
 	marketingConsent: <MarketingConsent />,
 };
+
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/pages/subscriptions-redemption/subscriptionsRedemption.tsx:26]: Cant see any changes here

**Review by paul-daniel-dempsey — APPROVED** [review]
Phew a lot taken out there, as discussed, might need Imogen to triple check. Only one tiny question I think, runs all fine. Great job!

**Comment by GHaberis** [comment]
@marialani I've put a Do Not Merge flag on, will explain more tomorrow.

**Pull request closed without merging**